Core/SaveState.cpp

```diff
@@ -248,8 +245,8 @@ namespace SaveState
 		std::mutex lock_;
 		std::thread compressThread_;
 
-		int base_;
-		int baseUsage_;
+		int base_ = -1;
+		int baseUsage_ = 0;
```

Comment:
Isn't this static?  Weird.

-[Unknown]

Reply:
Hm, yeah, weird indeed. Well I assumed this was the cause because after fixing it I can't repro the crash anymore, though it was a little bit random .. Dunno.